### ISSUE TITLE
Is there a reason to require symfony console v2?

### ISSUE THREAD

**Issue by ClaudiuCreanga** [richdynamix/magestead#51]
I am getting this error:

```
$ composer global require "richdynamix/magestead"
Changed current directory to /Users/Me/.composer
Using version ^2.1 for richdynamix/magestead
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for richdynamix/magestead ^2.1 -> satisfiable by richdynamix/magestead[2.1].
    - Conclusion: remove symfony/console v3.0.3
    - Conclusion: don't install symfony/console v3.0.3
    - richdynamix/magestead 2.1 requires symfony/console ^2.6 -> satisfiable by symfony/console[v2.6.0, v2.6.1, v2.6.10, v2.6.11, v2.6.12, v2.6.13, v2.6.2, v2.6.3, 
v2.6.4, v2.6.5, v2.6.6, v2.6.7, v2.6.8, v2.6.9, v2.7.0, v2.7.1, v2.7.10, v2.7.11, v2.7.12, v2.7.13, v2.7.2, v2.7.3, v2.7.4, v2.7.5, v2.7.6, v2.7.7, v2.7.8, v2.7.9, v2.8.0, v2.8.1, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6].
    - Can only install one of: symfony/console[v2.6.0, v3.0.3].
    - Can only install one of: symfony/console[v2.6.1, v3.0.3].
    - Can only install one of: symfony/console[v2.6.10, v3.0.3].
    - Can only install one of: symfony/console[v2.6.11, v3.0.3].
    - Can only install one of: symfony/console[v2.6.12, v3.0.3].
    - Can only install one of: symfony/console[v2.6.13, v3.0.3].
    - Can only install one of: symfony/console[v2.6.2, v3.0.3].
    - Can only install one of: symfony/console[v2.6.3, v3.0.3].
    - Can only install one of: symfony/console[v2.6.4, v3.0.3].
    - Can only install one of: symfony/console[v2.6.5, v3.0.3].
    - Can only install one of: symfony/console[v2.6.6, v3.0.3].
    - Can only install one of: symfony/console[v2.6.7, v3.0.3].
    - Can only install one of: symfony/console[v2.6.8, v3.0.3].
    - Can only install one of: symfony/console[v2.6.9, v3.0.3].
    - Can only install one of: symfony/console[v2.7.0, v3.0.3].
    - Can only install one of: symfony/console[v2.7.1, v3.0.3].
    - Can only install one of: symfony/console[v2.7.10, v3.0.3].
    - Can only install one of: symfony/console[v2.7.11, v3.0.3].
    - Can only install one of: symfony/console[v2.7.12, v3.0.3].
    - Can only install one of: symfony/console[v2.7.13, v3.0.3].
    - Can only install one of: symfony/console[v2.7.2, v3.0.3].
    - Can only install one of: symfony/console[v2.7.3, v3.0.3].
    - Can only install one of: symfony/console[v2.7.4, v3.0.3].
    - Can only install one of: symfony/console[v2.7.5, v3.0.3].
    - Can only install one of: symfony/console[v2.7.6, v3.0.3].
    - Can only install one of: symfony/console[v2.7.7, v3.0.3].
    - Can only install one of: symfony/console[v2.7.8, v3.0.3].
    - Can only install one of: symfony/console[v2.7.9, v3.0.3].
    - Can only install one of: symfony/console[v2.8.0, v3.0.3].
    - Can only install one of: symfony/console[v2.8.1, v3.0.3].
    - Can only install one of: symfony/console[v2.8.2, v3.0.3].
    - Can only install one of: symfony/console[v2.8.3, v3.0.3].
    - Can only install one of: symfony/console[v2.8.4, v3.0.3].
    - Can only install one of: symfony/console[v2.8.5, v3.0.3].
    - Can only install one of: symfony/console[v2.8.6, v3.0.3].
    - Installation request for symfony/console (locked at v3.0.3) -> satisfiable by symfony/console[v3.0.3].


Installation failed, reverting ./composer.json to its original content.
```

**Comment by richdynamix** [comment]
Hi Claudiu,

When I originally started writing magestead 2.0 the Symfony console 3.0 component was not yet released. I was also comfortable using 2.6 as it was a version I had previously used. That being said I can see now it is causing issues and should require an update.

I am not sure what the implications of simply updating the console to use the latest version is. I am going to assume that backwards compatibility breakage is expected based on the major bump.

I will investigate and get back to you. I will make this priority for the next release.

**Comment by richdynamix** [comment]
All Symfony components have been updated and are available in the latest release 2.1.1
